components/expansion_panels/TileExpansionPanel.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <v-row justify="center">
+    <v-expansion-panels id="TileExpansionPanels" tile>
```

Comment:
Please use id in one place for this element

Reply:
Done